packages/edit-site/src/components/header/more-menu/welcome-guide-menu-item.js

```diff
@@ -4,17 +4,13 @@
 import { __ } from '@wordpress/i18n';
 import { useDispatch } from '@wordpress/data';
 import { MenuItem } from '@wordpress/components';
-
-/**
- * Internal dependencies
- */
-import { store as editSiteStore } from '../../../store';
+import { store as preferencesStore } from '@wordpress/preferences';
 
 export default function WelcomeGuideMenuItem() {
```

Comment:
Why can't we use a `PreferenceToggleMenuItem` instead of `WelcomeGuideMenuItem`? Because we don't want to show a tick ✔️?

Reply:
Yep, it would also have the wrong aria role (`menuitemcheckbox` vs plain `menuitem`)